harvard-stellenbosch-university.csl

```diff
@@ -284,7 +286,8 @@
           <else-if type="chapter paper-conference entry-encyclopedia entry-dictionary" match="any">
             <text macro="title"/>
             <group delimiter=" ">
-              <text term="in" text-case="capitalize-first"/>
+              <!-- TODO: editors must be before title -->
```

Comment:
Any reason you're not doing that? This wouldn't seem hard to do.

Reply:
The definition to render the editor is in the `book-details` macro, so I can't just rearrange the elements. One would have to create a different `book-details` (without the editor-translator macro in the position as it is) specifically for use in book sections. But I didn't want to simply duplicate the macro with a few changes since that would lead to redundant / duplication of code. What I had in mind was that the `book-details` macro would have to be refactored so that it could be reused by both books and book-sections. (I've looked at it again and now think `book-details` should be split into `edition-volume` and `collection-details` so that `editor-translator` could be placed either in-between them or before both.)

Would it be preferable to open an issue to track this task?

Reply:
up to you -- having separate macros for editors for chapters/conference proceedings and elsewhere is very common in citation styles. It's not something I'd work on avoiding. If you want to wrap that into this PR, that's great, but if you want to wait for the next iteration, that's fine too. No need for an issue.